Source/LibationWinForms/Form1.VisibleBooks.cs

```diff
@@ -33,17 +33,11 @@ void setLiberatedVisibleMenuItem()
 				{
 					liberateVisibleToolStripMenuItem.Format(notLiberated);
 					liberateVisibleToolStripMenuItem.Enabled = true;
-
-					liberateVisible2ToolStripMenuItem.Format(notLiberated);
-					liberateVisible2ToolStripMenuItem.Enabled = true;
```

Comment:
Why were the liberateVisible2ToolStripMenuItem lines removed?

Reply:
Because I'm an idiot and didn't realize they were different from liberateVisibleToolStripMenuItem. Reverted.

Reply:
In fairness: I knew that stupid name would bite me one day. I'll change it soon

Reply:
renamed -- https://github.com/rmcrackan/Libation/commit/7987dfb81959c486c1b15c1d3787d2234aa7e134